.gitignore

```diff
@@ -5,6 +5,7 @@
 /coverage
 /build
 
+.env
```

Comment:
Why ignore `.env` and any env file used ?

Reply:
`.env` is the default environment, `client_id` is in this file

Reply:
Don't ignore this file, remove `client_id` or delete file if not used is just ok.

Reply:
ok